channels.yaml

```diff
@@ -148,6 +148,14 @@ releases:
       <<: *agentArgs-v1
       system-default-registry:
         type: string
+  - version: v1.21.10-rc1+k3s1
+    minChannelServerVersion: v2.6.0-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: *serverArgs-v2
+    agentArgs: &agentArgs-v2
```

Comment:
This can just be `agentArgs: *agentArgs-v2` since it is defined in v1.21.9 already